src/api/03-user-experience/07-autoplay.md

```diff
@@ -0,0 +1,225 @@
+</video>
+<script>
+  const videoElement = document.querySelector('video');
+
+  videoElement.play()
+    .catch(() => {
+      // Mute video and retry playback.
+      videoElement.muted = true;
+      videoElement.play()
+        .catch(() => {
+          // Can't autoplay with sound or muted.
+        })
+   })
+
+   /**
+    * Note: In older browsers the `play()` method may not
+    *       return a Promise. If you need to support
+    *       pre-2019 browsers, you could do:
+    *
+    * Promise.resolve(videoElement.play())
+    *   .catch(() => {
+    *     // Mute and try playing again.
+    *   })
+    *
+    * ... to wrap the non-Promise return values as Promises.
+</script>
+```
+
+### Unmute button
+
+Refresh this page and you'll likely notice the video now plays muted and there
+is a "Tap to unmute" button rendered over it.
+
+That's a custom element styled and positioned to render over the video. When
+the button is clicked, simple JavaScript runs and removes the `muted` attribute
+from the video and it continues playing with sound.
+
+This strategy combines the best of both worlds. Users are not going to get
+startled by sudden loud playback while the action necessary to achieve playback
+with sound is handily available.
+
+```html
+<section>
+  <video controls>
+    <source src="video.mp4" type="video/mp4">
+  </video>
+  <button>🔊</button>
+</section>
+
+<style>
+  /* Only basic layout, no fancy styling. */
+  section {
+    display: inline-block;
+    position: relative;
+  }
+  button {
+    font-size: 80px;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+  }
+</style>
+
+<script>
+  const videoElement = document.querySelector('video');
+  const unmuteButton = document.querySelector('button');
+
+  videoElement.play()
+    .catch(() => {
+      // Mute video and retry playback.
+      videoElement.muted = true;
+      videoElement.play()
+        .then(() => {
+          // When the unmute button is clicked,
+          // unmute video and remove the button.
+          unmuteButton.addEventListener('click', () => {
+            videoElement.muted = false;
+            unmuteButton.remove();
+          });
+        })
+        .catch(() => {
+          // Can't autoplay with sound or muted.
+        })
+   })
+</script>
+```
+
+## Engagement signals
+
+Browsers use various heuristics to determine whether automatic playback with
+sound is expected by the user or not. These heuristics differ between browsers,
+but here a few example use cases where autoplay with sound is likely:
+
+* When the newly loaded page is a result of internal navigation initiated by
+  the user, e.g. click, tap etc.
+* When the user has installed the PWA or added the site to their home screen.
+* When user engagement signals are strong, e.g. on video sites where the user
+  regularly watches the content.
+
+Find more in-depth information at the [Chrome Developers website].
+
+## What's Next?
+
+TBD
```

Comment:
Shall we remove TBD and link to Picture-in-Picture post?

Reply:
Updated in 281290836b44cdc445dea0ef7410d1082e9d8506.